ERCS/erc-xxxx.md

```diff
@@ -0,0 +1,244 @@
+| validationGasLimit            | QUANTITY       | Gas provided for the transaction account validation frame      |
+| paymasterValidationGasLimit   | QUANTITY       | Gas provided for the transaction paymaster validation frame    |
+| paymasterPostOpGasLimit       | QUANTITY       | Gas provided for the transaction paymaster `postOp` frame      |
+| callGasLimit                  | QUANTITY       | Gas provided for the transaction execution frame               |
+| accessList                    | OBJECT         | An EIP-2930 compatible Access List structure                   |
+| EIP-7702 authorizations (WIP) | ARRAY          | An EIP-7702 compatible list of contracts injected into EOAs    |
+| signature                     | DATA           | A signature of any kind used by Account to verify transaction  |
+
+### Changes to the JSON-RPC API
+
+#### Add RIP-7560 support for `eth_getTransactionReceipt`
+
+For an RIP-7560 transaction included in a block, return also the values specific to this transaction type
+in addition to all the existing fields.
+
+Parameters:
+
+1. DATA, 32 Bytes - hash of a transaction
+
+Returns:
+
+1. OBJECT - A transaction receipt object, or `null` when no receipt was found:
+
+Fields specific to an RIP-7560 transaction receipt:
+
+| Name                       | Type            | Description                                                                    |
+|----------------------------|-----------------|--------------------------------------------------------------------------------|
+| sender                     | DATA, 20 Bytes  | Address of the sender of this transaction                                      |
+| paymaster                  | DATA, 20 Bytes  | Address of the Paymaster if it is paying for the transaction, `null` otherwise |
+| deployer                   | DATA, 20 Bytes  | Address of the Deployer if it is included in the transaction, `null` otherwise |
+| senderCreationGasUsed      | QUANTITY        | The amount of gas actually used by the sender deployment frame                 |
+| senderValidationGasUsed    | QUANTITY        | The amount of gas actually used by the sender validation frame                 |
+| paymasterValidationGasUsed | QUANTITY        | The amount of gas actually used by the paymaster validation frame              |
+| executionGasUsed           | QUANTITY        | The amount of gas actually used by the execution frame                         |
+| postOpStatus               | QUANTITY        | 1 (success), 0 (failure), or `null` (did not run) status of the `postOp` frame |
+| postOpGasUsed              | QUANTITY        | The amount of gas actually used by the paymaster `postOp` frame                |
+| validationLogs             | ARRAY           | Array of log objects, which this transaction'S VALIDATION FRAME generated.     |
+| validationLogsBloom        | DATA, 256 Bytes | Array of log objects, which this transaction'S VALIDATION FRAME generated.     |
```

Comment:
not needed. the filter is used to select transactions, and that should include also TXs with validationLogs.
so it should be a standalone field.